src/core/InterleavedBuffer.js

```diff
@@ -92,6 +94,14 @@ InterleavedBuffer.prototype = {
 
 		return new this.constructor().copy( this );
 
+	},
+
+	onUpload: function ( callback ) {
```

Comment:
Does this function get called from anywhere? It would probably be good to show usage of this in a three.js example and/or use this function (and the callback) from GLTFLoader.js if the additions in that loader could e.g. delay its `.parse` `onLoad` callback from firing.